src/opm/parser/eclipse/EclipseState/Schedule/VFPInjTable.cpp

```diff
@@ -50,6 +50,147 @@ inline const Opm::DeckItem& getNonEmptyItem( const Opm::DeckRecord& record) {
 
 namespace Opm {
 
+VFPInjTable::VFPInjTable(int table_num,
+                         double datum_depth,
+                         FLO_TYPE flo_type,
+                         const std::vector<double>& flo_data,
+                         const std::vector<double>& thp_data,
+                         const array_type& data) {
+    m_table_num = table_num;
+    m_datum_depth = datum_depth;
+    m_flo_type = flo_type;
+    m_flo_data = flo_data;
+    m_thp_data = thp_data;
+
+    extents shape;
+    shape[0] = data.shape()[0];
+    shape[1] = data.shape()[1];
+    m_data.resize(shape);
+    m_data = data;
+
+    check();
+}
+
+
+
+VFPInjTable::VFPInjTable( const DeckKeyword& table, const UnitSystem& deck_unit_system) {
```

Comment:
It looks to me like this is essentially the same code as the init() method, but it's hard to tell if it is exactly that. Is it?